### PR TITLE
Sep viz ui

### PR DESCRIPTION
**Pull request by Saransh0905**
#71 Seperated tutorials 

I added 4 tutorials 

- viz_shapes.py ( for drawing shapes) 

- viz_buttons.py ( for buttons and text)

- viz_cube.py  (for cube and sliders to move cube)

- viz_ListBox.py (for Listbox)

### CODE REVIEW

**Comment by Saransh0905** [comment]
@skoudoro Please review these.

**Comment by skoudoro** [comment]
Hi @Saransh0905, Thank you for this.

There is still a lot of work:

- Can you fix all pep8 issues?
- All tutorials have the same title, Can you update the title, description, text, etc...
- Many functionalities are not exposed like the Listbox, etc... Please, Can you check all widgets/ui used and expose them clearly.

Thank you for looking into this

**Comment by Saransh0905** [comment]
@skoudoro I finally resolved pep8 issues 
Also added ListBox
If there's anything else let me know
Thanks

File: docs/tutorials/viz_cube.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+line_slider.set_visibility(True)
+
+###############################################################################
+# Set camera for better visualization
+
+show_manager.scene.reset_camera()
+show_manager.scene.set_camera(position=(0, 0, 150))
+show_manager.scene.reset_clipping_range()
+show_manager.scene.azimuth(30)
+show_manager.start()
```
Comment (skoudoro) [inline docs/tutorials/viz_cube.py]: can you replace this line by:

```python
interactive = False

if interactive:
    show_manager.start()

window.record(show_manager.scene, size=current_size, out_path="viz_button.png")
```

File: docs/tutorials/viz_listBox.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+current_size = (800, 800)
+show_manager = window.ShowManager(size=current_size,
+                                  title="DIPY UI ListBox_Example")
+
+show_manager.scene.add(listbox)
+show_manager.scene.add(Get_text)
+show_manager.scene.add(Set_text)
+show_manager.scene.add(Fury_text)
+show_manager.start()
```
Comment (skoudoro) [inline docs/tutorials/viz_listBox.py]: can you replace this line by:

```python
interactive = False

if interactive:
    show_manager.start()

window.record(show_manager.scene, size=current_size, out_path="viz_button.png")
```

File: docs/tutorials/viz_shapes.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+current_size = (800, 800)
+show_manager = window.ShowManager(size=current_size,
+                                  title="DIPY Shapes Example")
+
+show_manager.scene.add(rect)
+show_manager.scene.add(disk)
+show_manager.scene.add(ring)
+
+show_manager.start()
```
Comment (skoudoro) [inline docs/tutorials/viz_shapes.py]: can you replace this line by:

```python
interactive = False

if interactive:
    show_manager.start()

window.record(show_manager.scene, size=current_size, out_path="viz_button.png")
```

File: docs/tutorials/viz_buttons.py [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+panel having buttons with callbacks.
+
+First, some imports.
+"""
+from fury import ui, window
+from fury.data import read_viz_icons
+
+###############################################################################
+# Panel with buttons and text
+# ===========================
```
Comment (skoudoro) [inline docs/tutorials/viz_buttons.py]: remove the title and this line. Keep only the text below

File: docs/tutorials/viz_buttons.py [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    _button.next_icon()
+    i_ren.force_render()
+
+
+button_example.on_left_mouse_button_clicked = change_text_callback
+second_button_example.on_left_mouse_button_pressed = change_icon_callback
+
+###############################################################################
+# Show Manager
+# ==================================
```
Comment (skoudoro) [inline docs/tutorials/viz_buttons.py]: remove the title and this line

File: docs/tutorials/viz_buttons.py [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+# manager.
+
+current_size = (800, 800)
+show_manager = window.ShowManager(size=current_size,
+                                  title="DIPY Button Example")
+
+show_manager.scene.add(panel)
+
+###############################################################################
+# Default Visibility is True.
```
Comment (skoudoro) [inline docs/tutorials/viz_buttons.py]: Why? Can you explain it in a short sentence?

File: docs/tutorials/viz_listBox.py [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+First, a bunch of imports.
+"""
+from fury import ui, window
+
+###############################################################################
+# Create some text blocks that will be showm when
+# list elements will be selected
+
+Get_text = ui.TextBlock2D(text="Get", font_size=30, position=(500, 400))
```
Comment (skoudoro) [inline docs/tutorials/viz_listBox.py]: `Get` word can be confusing.  replace it with `welcome_text` and `text="Say welcome"`

File: docs/tutorials/viz_listBox.py [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+Set_text = ui.TextBlock2D(text="Set", font_size=30, position=(500, 400))
```
Comment (skoudoro) [inline docs/tutorials/viz_listBox.py]: `Set` word can be confusing.  replace it with `bye_text` and `text="Say Bye"`

File: docs/tutorials/viz_listBox.py [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+Fury_text = ui.TextBlock2D(text="Fury", font_size=30, position=(500, 400))
```
Comment (skoudoro) [inline docs/tutorials/viz_listBox.py]: `fury_text` instead of `Fury_text`

File: docs/tutorials/viz_listBox.py [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+    hide_all_examples()
+    element = example[values.index(listbox.selected[0])]
+    element.set_visibility(True)
+
+
+listbox.on_change = display_element
+
+###############################################################################
+# Show Manager
+# ==================================
```
Comment (skoudoro) [inline docs/tutorials/viz_listBox.py]: remove this line and title

File: docs/tutorials/viz_shapes.py [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+
+First, a bunch of imports.
+"""
+from fury.data import read_viz_icons, fetch_viz_icons
+
+from fury import ui, window
+
+###############################################################################
+# Shapes
+# ======
```
Comment (skoudoro) [inline docs/tutorials/viz_shapes.py]: Remove this line and title

File: docs/tutorials/viz_shapes.py [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+###############################################################################
+# Add an inner radius to make a ring.
+
+ring = ui.Disk2D(outer_radius=50, inner_radius=45,
+                 center=(500, 600), color=(0, 1, 1))
+
+
+###############################################################################
+# Show Manager
+# ==================================
```
Comment (skoudoro) [inline docs/tutorials/viz_shapes.py]: Remove this line and title

**Comment by Saransh0905** [comment]
@skoudoro I have done the changes you asked for
Please check and tell me if there anything else to do.

File: docs/tutorials/viz_shapes.py [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+==============
+Simple Shapes
+==============
+
+This example shows how to use the UI API. We will demonstrate how to draw
+some geometric shapes from DIPY UI elements.
+
+First, a bunch of imports.
+"""
+from fury.data import read_viz_icons, fetch_viz_icons
```
Comment (skoudoro) [inline docs/tutorials/viz_shapes.py]: To remove, this import is not used

File: docs/tutorials/viz_buttons.py [hunk 14]
```diff
@@ -0,0 +1,88 @@
...
+# Now that all the elements have been initialised, we add them to the show
+# manager.
+
+current_size = (800, 800)
+show_manager = window.ShowManager(size=current_size,
+                                  title="DIPY Button Example")
+
+show_manager.scene.add(panel)
+
+###############################################################################
```
Comment (skoudoro) [inline docs/tutorials/viz_buttons.py]: You can remove this line, it is not needed

File: docs/tutorials/viz_ui_slider.py [hunk 15]
```diff
@@ -0,0 +1,108 @@
...
+=====================
+Cube & Slider Control
+=====================
+
+This example shows how to use the UI API. We will demonstrate how to
+create a cube and control with sliders.
+
+First, some imports.
+"""
+from fury.data import read_viz_icons, fetch_viz_icons
```
Comment (skoudoro) [inline docs/tutorials/viz_ui_slider.py]: You can remove this line, it is not needed here

File: docs/tutorials/viz_buttons.py (hunk 14 above)
Comment (Saransh0905) [reply docs/tutorials/viz_buttons.py]: @skoudoro we have to add panel ui to the scene otherwise it showing a blank window

Comment (skoudoro) [reply docs/tutorials/viz_buttons.py]: I was talking of `###############################################################################` @Saransh0905. Not the `add(panel)`

**Comment by Saransh0905** [comment]
@skoudoro updated!

**Comment by skoudoro** [comment]
Thanks @Saransh0905, Looks good to me!

I will wait until tomorrow to see if anyone has more comments and then merge it.

**Pull request merged**